dev/plugins/hu.elte.txtuml.diagnostics/src/hu/elte/txtuml/diagnostics/animation/js/DiagnosticsServer.java

```diff
@@ -97,6 +98,27 @@ public void handle(HttpExchange exchange) throws IOException {
 		}
 
 	}
+	
+	private class DelayInputHandler implements HttpHandler {
+
+		@Override
+		public void handle(HttpExchange exchange) throws IOException {
+			// Set required headers
+			Headers headers = exchange.getResponseHeaders();
+			headers.add("Content-type", "application/json");
+			headers.add("Access-Control-Allow-Origin", "*");
+
+			// Build the payload
+			String response = "delayTime,["+ (diagnosticsPlugin.getDelay()/1000) +"]"; 
```

Comment:
Querying the delay timer does not work, one error can be found here:
The response is in JSON format, and thus should be as such: '{delayTime:5}'